external/requirejs/require.js

```diff
@@ -1664,13 +1671,12 @@ var requirejs, require, define;
 
                     //Join the path parts together, then figure out if baseUrl is needed.
                     url = syms.join('/');
-                    url += (ext || (/^data\:|\?/.test(url) || skipExt ? '' : '.js'));
+                    url += (ext || (/^data\:|^blob\:|\?/.test(url) || skipExt ? '' : '.js'));
```

Comment:
Does this change belong with the rest? It doesn't seem to be related to QUnit.

Reply:
@dmethvin This + Sinon changes is just running `npmcopy` on what we had. We now have `external` matching what's in `node_modules`. This required some minor changes in test code.

Reply:
Oh duh, i didn't notice this file was in `/external`  with the others. Carry on!